extension/js/agent/build/core.js

```diff
@@ -4,20 +4,13 @@ if (typeof window._ == "undefined") {
  var _ = this._;
 }
 
-// add jQuery if it's not there
-if (typeof window.$ == "undefined") {
-  // @include ../../lib/jquery/jquery.js
- var $ = this.$;
-}
-
-// add jQuery to window.__agent because Backbone requires it on the root
-this.$ = $ || window.$ || window.jQuery;
-var $ = this.$;
+// @include ../../lib/nanodom.js
+var nanodom = this.nanodom;
 
-// @include ../../lib/backbone-no-amd.js
+// @include ../../lib/backbone-events-standalone/backbone-events-standalone.js
```

Comment:
does this really make that much of a difference?  Not that backbone updates very often, but seems like this other lib would have a near 0% chance of upkeep.  why not just use `Backbone.Events` here?

Reply:
It makes difference. BB is 65Kb and Events is 9Kb

The agent.js build is injected into host page as big blob.  The bigger the agent file, more time is spent on parsing it, more memory is used.

But the greatest benefit is when debugging agent.js. Having so much unused code (BB.View, BB.Model/Collection, BB.Router) makes harder to search/scroll the code.

I would not bother in long term maintenance because the events are not used at all in agent.js. It is only used in tests. So in the short term it should be removed at all. I did not do in this PR because is already too big